10-JavaScript/js-in-the-browser.md

```diff
@@ -152,71 +97,32 @@ The DOM is effectively a tree, like the ones we've learned about in CS fundament
 
-## Using JavaScript and the DOM
-
-In JavaScript, the DOM is exposed through a set of objects and methods that provide access to the HTML structure of the webpage.
-
-We will find and practice methods for DOM manipulation that will allow us to do things like:
-- add a node (element) to the DOM
-- remove a node (element) from the DOM
-- change attributes of a node
-- enable interactions from a user, such as clicking a node (element)
-
-### Exercise
-
-The DOM is accessed through an object called `document`. Therefore, we will start DOM manipulation through this `document` object, since it is the broadest object that holds the DOM.
-
-1. In the Dev Tools Console, type into the console `document`. What do you get? When you expand it, what details do you see? What does it represent?
-1. Draw a representation of the DOM as a tree with a partner
+Run `document` in the console. What do you get?
 
-#### Finding an Existing Node and Modifying It
+## Browser-Supported Ways to Manipulate the DOM with JS
 
-We can find an existing node on the DOM in a sleuth of ways. We will explore one way:
-
-1. In the Dev Tools Console, enter into the console `document.getElementById('');` What do you get?
-1. In the Dev Tools Console, enter into the console `document.getElementById('js-lecture-target');` What do you get? What details do you see? What does it represent?
-1. Where is an element with the id `js-lecture-target` defined?
-
-Now let's try looking at these details and manipulating them.
+Logging to the console is alright, but the true power of JavaScript is that it can dynamically change the contents of the webpage.
 
-1. In the Dev Tools Console, enter in the following line:
-    ```javascript
-    document.getElementById('js-lecture-target').innerHTML;
-    ```
+Browsers have already defined a way (an interface) to represent, access, and modify the contents of a website. This interface is called **the DOM**. When we use this interface, we can dynamically change the page using JS.
 
-    What do you get back? What does it represent? Does it have the tags included?
-1. In the Dev Tools Console, enter in the following line:
-    ```javascript
-    document.getElementById('js-lecture-target').innerHTML = 'I can change website text with JavaScript!';
-    ```
+Until now our web applications have rendered _static_ HTML which defines a webpage. They have then sent it to the browser which in turn "builds the DOM" from that HTML.
 
-    What do you get back? What do you see on the website (HTML)?
-1. Refresh the page. Do you continue to see the changes that you made?
+When we use JavaScript to manipulate the DOM, we do not change the original HTML that was sent to the browser, instead we change the browser's internal representation of that same webpage. An important consequence of this distinction is that all of our changes disappear as soon as the browser forgets about that webpage (e.g. if we close the tab or the whole browser).
```

Comment:
I think it's important to call out refreshing as something that will lose their state.

```suggestion
When we use JavaScript to manipulate the DOM, we do not change the original HTML that was sent to the browser, instead we change the browser's internal representation of that same webpage. An important consequence of this distinction is that all of our changes disappear as soon as the browser forgets about that webpage (e.g. if we close the tab/browser or refresh the page).
```